.github/workflows/run-e2e-tests.yml

```diff
@@ -131,7 +90,7 @@ jobs:
         run: ./e2e/scripts/update_morse_config_from_secrets.sh
 
       - name: Run Morse E2E Tests with Service ID ${{ matrix.service_id }}
-        run: make test_e2e_evm_morse SERVICE_ID_OVERRIDE=${{ matrix.service_id }} WAIT_FOR_HYDRATOR=40
+        run: make test_e2e_evm_morse SERVICE_ID_OVERRIDE=${{ matrix.service_id }} WAIT_FOR_HYDRATOR=40 DOCKER_LOG=false
```

Comment:
As pointed out in my last comment, this will not actually override the `DOCKER_LOG=true` specified in the make target.